Added a follow-on question for on-call rotations. (#13)
tBaxter/questions-for-employers: README.md

This should have been in the job post, but if it wasn't, ask about it and think about how it fits with your lifestlye and commitments. Some folks are fine being road warriors, but if you've got kids or other commitments at home, it can be difficult.

**Would I need to be on call? How often?**   
Another nuts-and-bolts question, but on-call rotations can be tough. While you're asking, consider asking if being on-call comes with extra compensation.

> 

Reviewer: ```suggestion
```

This should have been in the job post, but if it wasn't, ask about it and think about how it fits with your lifestlye and commitments. Some folks are fine being road warriors, but if you've got kids or other commitments at home, it can be difficult.

**Would I need to be on call? How often?**   

> Another nuts-and-bolts question, but on-call rotations can be tough. While you're asking, consider asking if being on-call comes with extra compensation.

Reviewer: ```suggestion
Another nuts-and-bolts question, but on-call rotations can be tough. While you're asking, consider asking if being on-call comes with extra compensation.    
```

**Follow-ups**:  

> - If there is an on-call rotation, what is the training program like?**```

Reviewer: ```suggestion
- If there is an on-call rotation, what is the training program like?** On-call rotations can be intimidating for new developers, so there should be training programs or procedures they can speak to during the interview process.
```

**How many hours do people work in an average week? What time do people normally leave work? What time do they normally start? Do people work on the weekend?**

> Seriously, if the job can't be done in 40 hours, more or less, something is probably wrong.

Reviewer: ```suggestion
Seriously, unless there are unusual extenuating circumstances, the work should take 40 hours, more or less. If not, something is probably wrong.
```

**Would I need to be on call? How often?**   

> Another nuts-and-bolts question, but on-call rotations can be tough. While you're asking, consider asking of being on-call comes with extra compensation.

Reviewer: ```suggestion
Another nuts-and-bolts question, but on-call rotations can be tough. While you're asking, consider asking if being on-call comes with extra compensation.
```

> **If there is an on-call rotation, what is the training program like?**

Reviewer: ```suggestion
**Follow-ups**:  
- If there is an on-call rotation, what is the training program like?**```

> On-call rotations can be intimidating for developers new to the organization (or new to on-call rotations). Organizations should have training programs/procedures they can speak to during the interview procecss. These might include shadowing during another developer's rotation, a set of tutorials or videos, and written documentation.

Reviewer: ```suggestion On-call rotations can be intimidating for developers new to the organization (or new to on-call rotations). Organizations should have training programs/procedures they can speak to during the interview procecss. These might include shadowing during another developer's rotation, a set of tutorials or videos, and written documentation.
```
